pom.xml

```diff
@@ -191,6 +191,8 @@
 
     <!-- Latest release to be used by api-compatibility-check to check backwards compatibility of the Kie API. -->
     <version.org.kie.latestFinal.release>6.5.0.Final</version.org.kie.latestFinal.release>
+    <revapi.oldVersion>${version.org.kie.latestFinal.release}</revapi.oldVersion>
+    <revapi.newVersion>${project.version}</revapi.newVersion>
```

Comment:
I would suggest using more descriptive names, like `<revapi.kie.newVersion>` or maybe `<revapi.newKieVersion>`. I believe the `kie` is important to make it clear that the property expects a project (KIE) version.